crates/subspace-farmer/src/bin/subspace-farmer/commands/cluster/controller/farms.rs

```diff
@@ -32,8 +34,6 @@ use tracing::{error, info, trace, warn};
 type AddRemoveFuture<'a> =
     Pin<Box<dyn Future<Output = Option<(FarmIndex, oneshot::Receiver<()>, ClusterFarm)>> + 'a>>;
 
-pub(super) type FarmIndex = u16;
```

Comment:
This was meant to be a generic, cluster doesn't need to know what the type is, we could have used `u8`, but decided to allow for more than 256 farms in cluster mode. Why was this necessary?

Reply:
just move to `subspace_farmer::cluster::farmer`, because I want to use it in `ClusterFarmerIdentifyBroadcast`

Reply:
There should be no need to send it in there, it seems to be an artifact of ID derivation mechanism